### DISCUSSION TITLE
BasicAuth multiple users and passwords

### DISCUSSION THREAD

**Discussion by GektorUA** [tinyproxy/tinyproxy#482] [Q&A]
Does tinyproxy supports multiple users and passwords in one config file?

**Comment by rofl0r** [comment]
in basicauth.c, i see this comment:

```c
/*
 * Add entry to the basicauth list
 */
void basicauth_add (sblist *authlist,
        const char *user, const char *pass)
```
so yeah, it seems as if multiple user/pass combos can be stored. why didn't you just try it out ?

**Comment by w5rr** [comment]
I
